public/js/components/Editor.js

```diff
@@ -33,6 +67,7 @@ const Editor = React.createClass({
   },
 
   componentDidMount() {
+    this.cbPanels = {};
```

Comment:
I'm not sure if this is the best place for this.

*Why are you adding state?*
1. it is easier to clear the line widget if it is stored in on the component
2. it is easier to have a dictionary of widgets than assume there will only ever be one